public/css/desktop-style.css

```diff
@@ -179,591 +180,758 @@ body {
+  color: #f84e4e;
+  background-color: #090909 !important;
 }
 
 .dark .modal-footer .btn {
-    background: #191919;
+  background: #191919;
 }
 
 .dark .modal-footer .btn:hover {
-    background: rgba(0,0,0,0.8);
+  background: rgba(0, 0, 0, 0.8);
 }
 
 .dark hr {
-    color: black;
-    border: 1px #797979 solid;
-    background: black;
+  color: black;
+  border: 1px #797979 solid;
+  background: black;
 }
 
 .dark #note-family {
-    border-top: 1px #797979 solid;
+  border-top: 1px #797979 solid;
 }
 
 #dark-switcher:hover {
-    background: rgba(0,0,0,0.8);
-    color: #6ab0f3;
+  background: rgba(0, 0, 0, 0.8);
+  color: #6ab0f3;
 }
 
 img {
-    max-width: 100% !important;
+  max-width: 100% !important;
 }
 
 @media (max-width: 1000px) {
-    #user-email-heading {
-        display: none;
-    }
+  #user-email-heading {
+    display: none;
+  }
 }
 
 .dark #note-description-editor:focus {
-    outline: 1px solid rgba(0,0,0,.6) !important;
+  outline: 1px solid rgba(0, 0, 0, 0.6) !important;
+}
+
+/* Adding Table css  */
+/* 1> Table for Light mode */
+table {
+  width: 750px;
+  border-collapse: collapse;
+  margin: 50px auto;
+}
+
+/* Zebra striping */
+tr:nth-of-type(odd) {
+  background: rgb(239, 241, 122);
+}
+
+th {
+  background: #0b19968f;
+  color: white;
+  font-weight: bold;
+}
+
+td,
+th {
+  padding: 10px;
+  border: 1px solid #ccc;
+  text-align: left;
+  font-size: 18px;
+}
+
+/* 
+When view width gets very small,
+Making each row of table data as one block,which contains all the info of that particular entery.
+*/
+@media only screen and (max-width: 760px),
+  (min-device-width: 768px) and (max-device-width: 1024px) {
+  table {
+    width: 100%;
+  }
+
+  /* Force table to not be like tables anymore */
+  table,
+  thead,
+  tbody,
+  th,
+  td,
+  tr {
+    display: block;
+  }
+
+  /* Hide table headers (but not display: none;, for accessibility) */
+  thead tr {
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
+  }
+
+  tr {
+    border: 1px solid #ccc;
+  }
+
+  td {
+    /* Behave  like a "row" */
+    border: none;
+    border-bottom: 1px solid #eee;
+    position: relative;
+    padding-left: 50%;
+  }
+
+  td:before {
+    /* Now like a table header */
+    position: absolute;
+    /* Top/left values mimic padding */
+    top: 6px;
+    left: 6px;
+    width: 45%;
+    padding-right: 10px;
+    white-space: nowrap;
+    /* Label the data */
+    content: attr(data-column);
+    color: #000;
+    font-weight: bold;
+  }
+}
+
+/* 2> Table for dark mode */
+
+/* Zebra striping */
+.dark tr {
+  background: #8181818f;
+}
+
+.dark th {
+  background: rgb(56, 56, 55);
+  color: rgb(3, 129, 35);
+}
+
+.dark td,
+th {
+  padding: 10px;
+  border: 1px solid rgb(73, 75, 75);
+  text-align: left;
+  font-size: 18px;
+  color: rgb(123, 199, 218);
 }
+
+/* 
+When view width gets very small,
+Making each row of table data as one block,which contains all the info of that particular entery.
```

Comment:
This one is nice.
Just have the <th> different colors in this also, so that headings seems different.